CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # v 3.7.0 (?)
 Changes in this release:
 - Add support for ng containers.
+- Fix compatibility with updated ServiceInstance model -> new field:
+  - `desired_state_version` : The version of the desired state of a service instance. This version is incremented every time the desired state of the service is changing, i.e. the exported model.
```

Comment:
I don't understand this sentence.  Also, there is no logic here to actually increment this number?